src/db2/db2olap.tcl

```diff
@@ -1156,3 +1156,58 @@ if { $refresh_on } {
     do_tpch $dbname $user $password $scale_factor $RAISEERROR $VERBOSE $degree_of_parallel $total_querysets $myposition
 }}
 }
+
+proc delete_db2tpch {} {
```

Comment:
Wile I see that you're doing a "db2cmd db2 drop database" here, I could not find where we actually *create* the database, since I would have assumed that there is a similar "db2cmd db2 create database" command.

I do not see db2tcl having any in-built database creation logic either.

What am I missing?

Reply:
Saw this in the PR:

> To do this correctly we need to add the sqlecrea and the sqledrpd APIs to db2tcl and then we can both create and drop the schema from within HammerDB

But I'm still confused about how we actually create the database today.

Reply:
Right now we ask the user to create the Db2 database manually  - https://www.hammerdb.com/docs/ch04s02.html#d0e1176 and the build just does the load. I did not want to call OS commands as part of the schema build, as this will only work locally. However, as we have the drop schema functionality for all of the other databases this appears to be the best compromise so that it will also work for Db2 in some environments. 

The correct solution is to add the sqlecrea and the sqledrpd APIs to db2tcl and then this could be all done from within HammerDB - however this would take additional time to update the interface. I will create an issue to track this, once sqlecrea and sqledrpd are added to the interface then both the build and drop can be updated to use the correct interface.

Reply:
I created an issue in the db2tcl repo (https://github.com/memmertoIBM/db2tcl/issues/6) to add create/drop functionality.